src/components/parceires/index.js

```diff
@@ -58,30 +58,32 @@ const Parceires = () => {
         <section className='parceires'>
           <div className="swiper-slide">
             <ul>
-              {attributes.map((partners) =>
-                <li key={partners.nome}>
+              {attributes.map((partners) => {
```

Comment:
Trata-se de um único partner nessa função:

```suggestion
              {attributes.map((partner) => {
```